pegasus/sites.v3/hourofcode.com/public/learn/index.haml

```diff
@@ -125,7 +129,7 @@ social:
     });
     tutorialExplorerManager.renderToElement(document.getElementById('tutorials'));
 
-    var studentImage = "#{CDO.code_org_url('/images/learn/hoc2018_paint.png')}"; // hoc2018 override
```

Comment:
Can we delete these hoc2018_paint images now? Or are they still being used somewhere?

Reply:
Good point.  I notice we've kept a little library of these images in that directory.. there are few enough that it feels kind of cute to keep them?